apps/emqx_eviction_agent/test/emqx_eviction_agent_test_helpers.erl

```diff
@@ -81,7 +81,7 @@ start_cluster(NamesWithPorts, Apps, Env) ->
         NamesWithPorts
     ),
     Opts0 = [
-        {env, [{emqx, boot_modules, [broker, listeners]}] ++ Env},
+        {env, Env},
```

Comment:
Q: Honestly, it's not very clear what does "disabling the router module" means currently. The routing table are being updated irrespective of this setting, but publishing now will inevitably fail with router module disabled. Maybe it's time we reconsider what it means?

Reply:
the boot modules only affects tests.